index.html

```diff
@@ -5,310 +5,243 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
-    <link rel="stylesheet" href="./style.css" />
-    <link rel="stylesheet" href="./app.css">
+    <style>
+          @import url("https://fonts.googleapis.com/css2?family=Clicker+Script&family=Poppins:wght@400;500;600;700&display=swap");
+    </style>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="index.css" />
     <title>Setup Mobile First Portfolio</title>
 </head>
 <body>
     <main id="root">
-        <header class="navbar">
-            <div id="logo">
-                 <a href="#index"><img src="./images/MyLogo.png" alt="logo" /></a>
-             </div>
-             <nav>
-                   <div id="img">
-                       <img class="hamburger" src="./images/Union.png" alt="">
-                   </div>           
-               </nav>
-        </header>   
-        <section id="index">
-            <div id="container-def">
-            <h1>
-                I'm Enobong <br />
-                Glad to see you!
-            </h1>
-            <p>
-                I’m a software developer! I can help you build a product, feature
-                or website Look through some of my work and experience. If you like
-                what you see and have a project you need coded, don’t hesitate to
-                contact me. 
-            </p>
-            <br />
-            <h4>LET'S CONNECT</h4>
-            <div id="icon-section">
-                <ul>
-                    <li>
+    <header class="navbar">
+        <a href="#index"><img src="./images/MyLogo.png" alt="logo" /></a>
+        <nav class="deskNav">
+            <ul class="navbar-items">
+                <li><a href="#">Portfolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+            <img class="hamburg" src="pics/hamburg.svg" alt="hamburg" />
+        </nav>
+    </header>
```

Comment:
- [ ] Where is your mobile navigation? You need to have the hamburger here and hide it with CSS on a larger screen but display it back on small screens. 
![miss](https://user-images.githubusercontent.com/99656687/221350400-40f5e14f-cd17-49ef-98d7-dfbff83acfcb.png)